tick/optim/model/hawkes_utils.py

```diff
@@ -0,0 +1,111 @@
+# License: BSD 3 clause
```

Comment:
This file is a duplication of tick/optim/model/hawkes_utils.py and should be deleted